full_stack_projects/aa_approved_full_stack_projects.md

```diff
@@ -0,0 +1,222 @@
+# a/A Approved list of Apps
+
+## Apps requiring AWS:
+- All Trails - _Google Maps required_
+- AirBnB - _Google Maps required_
+- HipCamp - _Google Maps required_
+- Yelp - _Google Maps required_
+- Open Table - _Google Maps required_
+- Streeteasy - _Google Maps required_
+- Zillow - _Google Maps required_
+- 500px
+- Etsy
+- Amazon
+- Eventbrite
+- Flickr
+- Goodreads
+- Medium
+- LinkedIn
+- Pinterest
+- Shopping site
```

Comment:
Limit this list to specific apps and maybe make a note elsewhere about the wide variety of e-commerce applications available to clone

Reply:
Updating in the next push